tests/functional/test_download.py

```diff
@@ -666,7 +666,9 @@ def test_download_exit_status_code_when_blank_requirements_file(script):
     Test download exit status code when blank requirements file specified
     """
     script.scratch_path.joinpath("blank.txt").write_text("\n")
-    script.pip('download', '-r', 'blank.txt')
+    result = script.pip('download', '-r', 'blank.txt', expect_error=True)
+    print(result)
+    assert result.returncode == ERROR
```

Comment:
```suggestion
    assert result.returncode == ERROR, str(result)
```

Reply:
This change is now reverted as per https://github.com/pypa/pip/pull/8265#discussion_r429269044